NoSoft-SA/nspack
# Feature/user validate po summary

Summary of this change:
Added a view summarising the validation changes made on a PO receipt.
Additionally, fixed a typo in the PI documentation.



CHANGELOG entry:

### Added
- A new view `EDI | Receipts | (via Action menu) validation summary` which is only shown if `edi_in_transactions.validated_by_user` is true.
- Routing, views and UI-rules required for `validation summary`. 
### Changed
- Renamed the previous  `EDI | Receipts | (via Action menu) validation summary` menu item to `pallet diff summary`.
### Fixed
- `EDI | Config | Out flow doc | PI` bug, `undefined method << ...`

**How to test this code via the UI:**
Manually validate a PO-in file, (requires `depot_receipts_require_user_validation` client rule and `user_can_manually_validate` user permission) and see if the summary displays correctly for:
- added,
- removed and 
- changed pallet records.
- As well as a changed header.
Finally, manually validate a PO-in file without any changes and confirm the summary page displays correctly.
